pkgs/applications/networking/sniffers/qtwirediff/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, qtbase
+, qmake
+, qtwayland
+, wrapQtAppsHook
+}:
+
+
+stdenv.mkDerivation {
+  pname = "qtwirediff";
+  version = "unstable-2022-01-15";
```

Comment:
It seems upstream has moved forward and is using Qt 6, do you think it would make sense to upgrade it?

Reply:
Yes, just updated it, I will commit it in a few minutes.